app/viewmodels/checkAnswers/sections/consignee/ConsigneeExportEoriSummary.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.checkAnswers.sections.consignee
+
+import controllers.sections.consignee.routes
+import models.CheckMode
+import models.requests.DataRequest
+import pages.sections.consignee.ConsigneeExportEoriPage
+import play.api.i18n.Messages
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object ConsigneeExportEoriSummary  {
+
+  def row(showActionLinks: Boolean)(implicit request: DataRequest[_],  messages: Messages): Option[SummaryListRow] =
+    request.userAnswers.get(ConsigneeExportEoriPage).map {
+      answer =>
+
+        SummaryListRowViewModel(
+          key     = "consigneeExportEori.checkYourAnswersLabel",
+          value   = ValueViewModel(HtmlFormat.escape(answer).toString),
+          actions = if (!showActionLinks) Seq() else Seq(
+            ActionItemViewModel(
+              content = "site.change",
+              href = routes.ConsigneeExportEoriController.onPageLoad(request.ern, request.draftId, CheckMode).url,
+              id = "changeConsigneeExportEori"
+            ).withVisuallyHiddenText(messages("consigneeExportEori.change.hidden"))
+          )
+        )
+    }
+}
```

Comment:
Could you add some tests for this please?

Reply:
Added missing tests https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/245/commits/46a138e7450b99004dcef7a3623cd0257426371a